### DISCUSSION TITLE
Hi, can you contact me?

### DISCUSSION THREAD

**Discussion by Ricky1966** [Donkie/Spoolman#121] [General]
Hi, my name is Riccardo Bramati, I'm writing from Italy. I've started a project to implement an ESP32 and one or more RFID Sensors for read and write NFC stickly Tags. My goal is to use that tags to store some infos about spool, as length and weight, printing temperature, material and color of filament. For now, I've implemented a web server over ESP32, with a UI for read tags data (I'm working to omplements others features like write, erase, cancel and format. Seems we have same ideas bit with different way, if you are intresting, we may collaborate to making a join of this two work and give to a 3d printer user, a more intresting sustem for store spools and use as well as possible.
Tell me what do you think about, this is my email address is :
riccardo (dot) bramati (@) gmail (dot) com
Sorry for I've write address in this way, but you know how is dangerous leave an email address clearly.
Also sorry for my English, but itn't my language.
Waiting for your answer, my best regards
Ricky1966

**Comment by Donkie** [comment]
Hi! That sounds interesting. I unfortunately don't really have time to branch out to things like this at the moment. But this is an open source repository so feel free to suggest improvements to help achieving your goals.

**Reply by Ricky1966** [reply]
Ok tks for you kindness. 
I'll look inside your repository. I'm sure I will find somethink intresting for me.
If you want to warch my jjob looking for Ricky1966 and NFC-Spools on Github
